docs/community/maintainer/pr-review.md

```diff
@@ -0,0 +1,16 @@
+# Pull Request Review Policy
+
+This document outlines the review policy for pull requests in the Trivy project.
+
+## Core Principles
+
+### 1. All Changes Through Pull Requests
+All changes to the `main` branch must be made through pull requests. Direct commits to `main` are not allowed.
+
+### 2. Required Approvals
+Every pull request requires approval from at least one CODEOWNER before merging.
```

Comment:
what about PR created by the only code owner? in this case any other maintainer should approve